src/core/lib/slice/slice_buffer.cc

```diff
@@ -203,6 +238,36 @@ void grpc_slice_buffer_reset_and_unref(grpc_slice_buffer* sb) {
   }
 }
 
+void _helper_grpc_slice_buffer_move_into_empty_buf(grpc_slice_buffer* src,
+                                                   grpc_slice_buffer* dst) {
+  if (src->base_slices == src->inlined) {
+    // If dst not inlined but is empty, it was at some point grown via
```

Comment:
I don't understand this comment.  How do we know that `dst` was grown at some point?